config/crd/bases/infrastructure.cluster.x-k8s.io_azuremachinepools.yaml

```diff
@@ -1380,6 +1380,14 @@ spec:
                   meaning that the node can be drained without any time limitations.
                   NOTE: NodeDrainTimeout is different from `kubectl drain --timeout`'
                 type: string
+              orchestrationMode:
+                default: Uniform
```

Comment:
I thought we discussed that we'd default to Flexible going forward and convert previous versions setting Uniform on them. I believe this line will default to Uniform if not specified.

Reply:
When I discussed with @CecileRobertMichon, we thought it was safer to leave the default as `Uniform` and require users to opt in. Especially since VMSS Flex support requires out-of-tree cloud provider and Kubernetes >= 1.26.

Reply:
Makes sense. Thank you for the explanation.